README.md

```diff
@@ -1,58 +1,145 @@
 # External DNS
 [![Build Status](https://travis-ci.org/kubernetes-incubator/external-dns.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/external-dns)
 [![Coverage Status](https://coveralls.io/repos/github/kubernetes-incubator/external-dns/badge.svg?branch=master)](https://coveralls.io/github/kubernetes-incubator/external-dns?branch=master)
-[![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-incubator/external-dns)](https://goreportcard.com/report/github.com/kubernetes-incubator/external-dns)
 
-`external-dns` synchronizes external DNS servers (e.g. Google CloudDNS) with exposed Kubernetes Services and Ingresses.
+ExternalDNS synchronizes exposed Services and Ingresses with cloud DNS providers.
 
-This is a new Kubernetes Incubator project and will incorporate features from the following existing projects:
+# Motivation
 
-* [Kops DNS Controller](https://github.com/kubernetes/kops/tree/master/dns-controller)
-* [Mate](https://github.com/zalando-incubator/mate)
-* [wearemolecule/route53-kubernetes](https://github.com/wearemolecule/route53-kubernetes)
+Inspired by Kubernetes' cluster-internal [DNS server](https://github.com/kubernetes/dns) ExternalDNS intends to make Kubernetes resources discoverable via public DNS servers. Similarly to KubeDNS it retrieves a list of resources from the Kubernetes API, such as Services and Ingresses, to determine a desired list of DNS records. However, unlike KubeDNS it's not a DNS server itself but merely configures other DNS providers accordingly, e.g. AWS Route53 or Google CloudDNS.
 
-## Current Status
+In a broader sense, it allows you to control DNS records dynamically via Kubernetes resources in a DNS provider agnostic way.
 
-The project just started and isn't in a usable state as of now. The current roadmap looks like the following.
+# Quickstart
 
-* Create an MVP that allows managing DNS names for Service resources via Google CloudDNS with the official annotation set. (Done)
-* Add support for Ingress and Node resources as well as AWS Route53.
-* Add support for the annotation semantics of the three parent projects so that `external-dns` becomes a drop-in replacement for them.
-* Switch from regular sync-only to watch and other advanced topics.
+ExternalDNS' current release is `v0.1.0-beta.0` which allows to keep a managed zone in Google's [CloudDNS](https://cloud.google.com/dns/docs/) service synchronized with Services of `type=LoadBalancer` in your cluster.
```

Comment:
I agree, but I want to be explicit about the versions here. I'm going to change it to the more general `v0.1` as this is what I wanted to say in the first place. Think of `v0.1` as the name of a milestone that I reference here. Is that something we can do until we get to a more stable release?